pegasus/sites.v3/code.org/views/carousel_not_hacked_video.haml

```diff
@@ -0,0 +1,27 @@
+- video_not_hacked_index = 7
+- video_not_hacked_id = ["not_hacked_intro", "not_hacked_passwords","not_hacked_auth", "not_hacked_phising", "not_hacked_wifi", "not_hacked_messaging", "not_hacked_updates"]
+- video_not_hacked = ["4J6FNRft9pQ", "qLMIK5YVgfA", "l9mEd8LOMl8", "YfiM8rXg3ug", "E3imvkP3IC0",  "IxjWR_uexOc", "goMnkhKBOHY"]
+
+
+
+.carousel-wrapper#not_hacked_works_collection
+  .carousel.not_hacked_works_collection
+    .slides
+      - video_not_hacked_index.times do |index|
+        .slide
+          .action-block.action-block--one-col.flex-space-between
+            .video__container__without__caption
+              = view :display_video_thumbnail, id: video_not_hacked_id[index], video_code: video_not_hacked[index], play_button: 'center', letterbox: 'false'
+  
+            
+  .pagination
+    #not_hacked_works_collection_pagination
+  .dd__buttons
+    %button.carousel__prev#prev_6
+      %img{src: "/images/carousel/prev.svg", alt: "previous"}
+
+    %button.carousel__next#next_6
+      %img{src: "/images/carousel/next.svg", alt: "next"}
+
+
+
```

Comment:
Nit: extra whitespace — there can just be one line of whitespace at the end of the file